Source/ParameterEncoding.swift

```diff
@@ -101,19 +109,16 @@ public struct URLEncoding: ParameterEncoding {
             case .numeric:
                 return value ? "1" : "0"
             case .literal:
-                return value ? "true" : "false"
+                return value.description
             }
         }
     }
 
     // MARK: Properties
 
-    /// Returns a default `URLEncoding` instance.
+    /// Returns a default `URLEncoding` instance with a `.methodDependent` destination.
```

Comment:
I'm not sure about this one. Perhaps keeping `default` and adding an `httpBody` convenience property?

Reply:
I am not understanding your comment. I need more context. What is the issue about?